lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc

```diff
@@ -628,8 +628,12 @@ function dosomething_campaign_get_scholarship($nid) {
  *  Formatted time (e.g. "4 days left")
  */
 function dosomething_campaign_get_hot_time_left($node, &$vars = NULL) {
-  $today = isset($vars['hm_today']) ? $vars['hm_today'] : dosomething_campaign_set_est_timezone(new DateTime());
-  $end_date = $vars['hm_end_date'];
+  if (!isset($node) && !isset($vars)) {
+    return;
+  }
+  $timing = dosomething_campaign_get_goals_times($vars ?: $node);
```

Comment:
this should already be set in the `vars` array (except in the instance of cron, so this can be run only if you need a fallback